### PR TITLE
YAML Editor Automation

### PR DESCRIPTION
**Pull request by gajanan-more**
**Fix:** https://issues.redhat.com/browse/ODC-5476

**Problem:** Automation to create pipeline using YAML Editor

**Solution:** This script will create pipeline using YAML View on Pipeline Builder page

**Test Setup:**
1. Run **_yarn run test-cypress-pipelines_** in frontend
2. Add **_@smoke_** above every scenario from file **_create-pipeline.feature_** 
3. Execute feature file: **frontend/packages/pipelines-plugin/integration-tests/features/ui-validation-with-yaml/create-pipeline.feature**

**Note: There are some problems with cy.login(), please comment the cy.login() command from _packages/dev-console/integration-tests/support/commands_ and _packages/pipelines-plugin/integration-tests/support/commands_.**

**Test Execution Browser:** Google Chrome

**Screenshot:**
Screenshot from 2021-03-11 17-36-21

### CODE REVIEW

**Comment by sanketpathak** [comment]
The tests ran successfully on local
Screenshot from 2021-03-12 15-46-08
/lgtm

**Comment by makambalaji** [comment]
@gajanan-more - Please execute `Yarn run gherkin-lint for the specific file to check the lint issues`
Observed below lint issues
image

**Comment by gajanan-more** [comment]
> @gajanan-more - Please execute `Yarn run gherkin-lint for the specific file to check the lint issues`
> Observed below lint issues
> image

Updated @makambalaji

**Review by makambalaji — COMMENTED** [review]
Please updated page object as well

File: frontend/packages/pipelines-plugin/integration-tests/support/page-objects/pipelines-po.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ export const pipelineBuilderPO = {
     yamlView: '#form-radiobutton-editorType-yaml-field',
   },
   formView: {
+    switchToFormView: 'id="form-radiobutton-editorType-form-field"',
```
[the remark this answers is not in the document]
Comment (gajanan-more) [reply frontend/packages/pipelines-plugin/integration-tests/support/page-objects/pipelines-po.ts]: Updated

**Pull request merged**